.eslintrc

```diff
@@ -4,6 +4,7 @@
     "es6": true,
```

Comment:
@ScionOfBytes I don't how I got the older file cloned. Let me update to `standard`.